### DISCUSSION TITLE
Don't <Link> to a table if it's the currently active one

### DISCUSSION THREAD

**Discussion by saltcod** [supabase/supabase#17367] [Contribute to Supabase]
In the list of tables, we shouldn't <Link> to the one that's currently active. It causes existing filters/sorts to be removed: 

To reproduce:
![screenshot-2023-09-12-at-11 16 41](https://github.com/supabase/supabase/assets/105593/1a085e1d-19e4-4492-ae64-d735bd4bd7aa)

Solution would be to use a <Link> to link to all tables except the currently active one. For that, we should just render the table name.

**Comment by saltcod** [comment]
Coincidentally, we received the opposite feedback the same day I opened this discussion:

> Hey! Can you make it so that clicking on an already selected table name in the Dashboard Table Editor serves as another way to do the table view reload? It seems intuitive to me, I reflexively do it.

Any thoughts on this? How should the already-active table link behave?

**Reply by faizanraso** [reply]
Hey @saltcod, to me it makes sense to reset the table entirely, so this would include clearing the existing filters/sorting, as well as refresh it to include any new data, policy etc.

**Comment by Dosbodoke** [comment]
@saltcod I think clicking on the already selected table shouldn't remove the filters, instead, we could add a clear filters button. Clicking on other tables should reset

IDK how the supabase team approach design decisions, but there is improvement to be done here
